### DISCUSSION TITLE
GPT 4 Application code

### DISCUSSION THREAD

**Discussion by JenipherJosiah** [xtekky/gpt4free#1361] [Q&A] [answered]
Hi, I have started to program a software application to android where it contains video input which contains an idea so AI recognizes and provide solutions so I have been gathering information using AI tools and AI resources like [AI Chat GPT](https://aichatgpt.one/), So I need alternative options as that to find best codings that suit my app.

**Comment by eminsk** [comment, marked as answer]
You can use various AI tools and resources to create an Android app that recognizes ideas in videos and provides solutions. Below are a few options that may be useful:
TensorFlow Lite: TensorFlow Lite provides examples of video classification that can be useful in video content recognition. This can be useful for training a model to recognize ideas in videos
ML Kit from Google: ML Kit allows you to detect and track objects in successive video frames. This can be useful for detecting objects related to ideas in a video
Perpetio: Perpetio specializes in fitness apps and uses machine learning technologies to create image recognition apps. Their experience can be useful when creating an application for recognizing ideas in videos
AI camera apps: There are also a number of AI camera apps for Android that can provide scene and face recognition features that can be useful when working with videos
